### ISSUE TITLE
flow field in learn2reg using Pytorch implementation

### ISSUE THREAD

**Issue by hassaanmahmood** [voxelmorph/voxelmorph#165]
I am trying to run learn2reg using pytorch implementation of voxelmorph. But I am not getting flow field as  from TensorFlow implementation.

1- gradient loss started from zero. Why that so?
 after 25 iteration Grad_loss started to increase.
this is the output of last iteration
`999,loss 0.003818,recon_loss 0.003319, grad_loss 0.049900`
which seems not bad.i think.

2- Can you please have a look, What can be the problem here. when I visualize flow  using squeeze its not as expected.

[lern2reg_pytorch.pdf](https://github.com/voxelmorph/voxelmorph/files/4402104/lern2reg_pytorch.pdf)

**Comment by hassaanmahmood** [comment]
Or may be it is just the number of iteration that I need to increase!

**Comment by adalca** [comment]
@hassaanmahmood I'm not sure I understand the questions here -- there is not requirement for the gradient loss to decrease, it's just the *overall* loss that has to decrease. 

For the last figure you showed, while I agree that it's peculiar that the flow has limited vertical signal, I would encourage you to run the training much longer before worrying about it for now.
